azavea/pwd-fishway
# Add animated background

## Overview
This PR adds a new Background component that contains a wave animation and changes state based on which tab is selected.

Connects #66

### Demo
![fishway_animated_background_sm](https://user-images.githubusercontent.com/5672295/58333528-9f11e180-7e0b-11e9-9e0e-792188b76d18.gif)


### Notes
- [react-lottie](https://www.npmjs.com/package/react-lottie) is now added to our packages (and it's great!)
- The Navbar is currently hard to see when the "sky" is in view. I'll handle theming it in #83

## Testing Instructions
* `git pull`
* Run `./scripts/update` 
* You should see a new background and water animation
* Enter the app, note that the background slides away
* Click on different pages in different orders to ensure that the animation doesn't look awkward. This being said, some finessing may still need to be done once we have completed work in #88
